manifests/foreman_proxy.pp

```diff
@@ -82,25 +88,34 @@
   if $deploy {
 
     certs::keypair { 'foreman_proxy':
-      key_pair   => Cert[$proxy_cert_name],
-      key_file   => $proxy_key,
-      manage_key => true,
-      key_owner  => 'foreman-proxy',
-      key_mode   => '0400',
-      key_group  => $group,
-      cert_file  => $proxy_cert,
+      key_pair    => Cert[$proxy_cert_name],
+      key_file    => $proxy_key,
+      manage_key  => true,
+      key_owner   => 'root',
+      key_mode    => '0640',
```

Comment:
I think we should keep this at `0400` as it was previously. There is not a situation I can perceive where this should be writable and can prevent accidental debug errors where a file gets edited by someone opening it.

Reply:
I disagree. 0400 is bad because the service itself can modify the file. Remember the owner of a file can just chmod 600. By using the group, we can prevent it from doing so. I'd also be fine with 0440, but the critical change here is that we prevent foreman-proxy from changing the config.

Reply:
I vote `0440` then.